wages_challenge.sh

```diff
@@ -0,0 +1,28 @@
+#Gets unique combinations of gender and yearsExperience, sorting by gender and then yearsExperience
+echo "Unique combos of gender and yearsExp, sorted:"
+cat wages.csv | grep -v "gender" | cut -d ',' -f 1,2 | sed 's/,/ /g' | sort -k1,1 -k2n | uniq
+
+#---Start of challenge 2---
+#Returns highest earner's info
+echo "Highest Earner:"
+cat wages.csv | grep -v "gender" | tr "," " " | sort -n -k4 | tail -n 1
+
+#Returns the lowest earners info
+echo "Lowest Earner:"
+cat wages.csv | grep -v "gender" | tr "," " " | sort -n -k4 | head -n 1
+
+#Returns the number of females in the top ten earners
+echo "Number of Females in the Top 10:"
+cat wages.csv | grep -v "gender" | tr "," " " | sort -n -k4 | tail -n 10 | grep -c female
+#---End of Challenge 2---
+
+#---Start of challenge 3---
+#Returns minimum wage for 12 years of education
+minwage_12=$(cat wages.csv | grep -v "gender" | tr "," " " | grep -w "12" | sort -n -k4 | head -n 1 | cut -d " " -f4)
```

Comment:
Close. You are finding lines with both 12 years schooling and 12 years experience. Use the cut tool to remove the years experience column before grep -w "12".